tests/conftest.py

```diff
@@ -223,7 +246,7 @@ def factory(
 
     def finalizer():
         """
-        Delete the Ceph block pool
+        Delete the storageclass
```

Comment:
@fbalak if we are using default SC, we dont need Delete Storageclass.. right ?

Reply:
Yes, but this is generic factory fixture that is not used for default storageclass. I made this change just to fix a docstring that I noticed is wrong.

Reply:
ok